1-org/envs/shared/org_policy.tf

```diff
@@ -177,3 +177,66 @@ resource "google_access_context_manager_access_policy" "access_policy" {
   parent = "organizations/${local.org_id}"
   title  = "default policy"
 }
+
+
+module "restrict_procotol_fowarding" {
+  source          = "terraform-google-modules/org-policy/google"
+  version         = "~> 5.1"
+  organization_id = local.organization_id
+  folder_id       = local.folder_id
+  policy_for      = local.policy_for
+  policy_type     = "list"
+  allow           = ["internal"]
+  enforce         = "true"
+  constraint      = "constraints/compute.restrictProtocolForwardingCreationForTypes"
+}
+
+module "disable_vpc_external_ipv6" {
+  source          = "terraform-google-modules/org-policy/google"
+  version         = "~> 5.1"
+  organization_id = local.organization_id
+  folder_id       = local.folder_id
+  policy_for      = local.policy_for
+  policy_type     = "boolean"
+  enforce         = "true"
+  constraint      = "constraints/compute.disableVpcExternalIpv6"
+}
+
+
+module "disable_service_account_key_upload" {
+  source          = "terraform-google-modules/org-policy/google"
+  version         = "~> 5.1"
+  organization_id = local.organization_id
+  folder_id       = local.folder_id
+  policy_for      = local.policy_for
+  policy_type     = "boolean"
+  enforce         = "true"
+  constraint      = "constraints/iam.disableServiceAccountKeyUpload"
+}
+
+data "google_organization" "org" {
+  organization = var.org_id
+}
+
+module "domain_restricted_contacts" {
+  source          = "terraform-google-modules/org-policy/google"
+  version         = "~> 5.1"
+  organization_id = local.organization_id
+  folder_id       = local.folder_id
+  policy_for      = local.policy_for
+  policy_type     = "list"
+  allow           = [data.google_organization.org.directory_customer_id]
+  enforce         = "true"
```

Comment:
this one should not have  ` enforce = "true"` 
https://github.com/terraform-google-modules/terraform-google-org-policy/blob/master/variables.tf#L39-L43

It is not Boolean and it is not deny all / allow all